instrumentation/action_mailer/Rakefile

```diff
@@ -15,6 +15,7 @@ Rake::TestTask.new :test do |t|
   t.libs << 'test'
   t.libs << 'lib'
   t.test_files = FileList['test/**/*_test.rb']
+  t.warning = false
```

Comment:
I took the liberty of adding this, when adding the new tests that do ActionMailer deliveries the `mail` gems generates lots of warnings. I saw other instrumentation libraries already had it.